relays/table_transform.ts

```diff
@@ -23,10 +24,10 @@ export default function(msg: Message): void|Message|Message[]|Promise<Message> {
 	let rounds = roundsArray(msg.data.stage.rounds);
 
 	let data = {
-		header: [NUMBER, NAME, TOP].concat(rounds),
-		data: msg.data.ranking.map(function(rank) {
+		header: [RANK, NUMBER, NAME, TOP].concat(rounds),
+		data: msg.data.ranking.map(function(rank, index) {

```

Comment:
If each line has it's own rank, then if there are two teams with the same rank (say, 16), and the "page break" happens between those two teams, then there is no problem. The one screen will end with 16 and the next will start with 16. That seems fine to me.